### PR TITLE
Add stub react app for voter-dapp

### PR DESCRIPTION
**Pull request by ptare**
One thing different vs. sponsor-dapp is that addition of the service worker file (create-react-app added it automatically). I'm not sure whether we want that or not, we can also delete it.

Signed-off-by: Prasad Tare <prasad.tare@gmail.com>

### CODE REVIEW

File: voter-dapp/src/logo.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (mrice32) [inline voter-dapp/src/logo.svg]: What is this logo for?

File: voter-dapp/src/App.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./App.css";
+
+function App() {
+  return <div className="App">Voter DApp</div>;
```
Comment (mrice32) [inline voter-dapp/src/App.js]: nit: I think we prefer the styling `dApp` for external viewing.

File: voter-dapp/public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "UMA Voter DApp",
+  "name": "UMA Voter DApp",
+  "icons": [
+    {
+      "src": "favicon.ico",
```
Comment (mrice32) [inline voter-dapp/public/manifest.json]: nit: did you swap out the favicon for the UMA one that's in the `sponsor-dapp` directory?

File: voter-dapp/public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
```
Comment (mrice32) [inline voter-dapp/public/manifest.json]: What are these used for? Should they be set to the UMA colors?

File: voter-dapp/README.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
```
Comment (mrice32) [inline voter-dapp/README.md]: Can we tear most of this documentation out? I think it's kinda overwhelming. IMO, it'd be nice to just leave the documentation for the commands we use in our normal dev flow:
- `npm start`
- `CI=true npm test`
- `npm run build`

And then a link to the react documentation website.

File: package.json [hunk 6]
```diff
@@ -25,7 +25,7 @@
     "@sendgrid/mail": "^6.4.0",
     "bignumber.js": "^8.0.1",
     "dotenv": "^6.2.0",
-    "eslint": "5.12.0",
+    "eslint": "5.16.0",
```
Comment (mrice32) [inline package.json]: Was this on purpose? (Totally cool if it was, just wanted to double check)

File: voter-dapp/src/logo.svg (hunk 1 above)
Comment (ptare) [reply voter-dapp/src/logo.svg]: Not a clue haha, deleted it.

File: voter-dapp/src/App.js (hunk 2 above)
Comment (ptare) [reply voter-dapp/src/App.js]: Done.

File: voter-dapp/public/manifest.json (hunk 3 above)
Comment (ptare) [reply voter-dapp/public/manifest.json]: Yeah I copied it over from there.

$ diff ../sponsor-dapp/public/favicon.ico public/favicon.ico

File: voter-dapp/public/manifest.json (hunk 4 above)
Comment (ptare) [reply voter-dapp/public/manifest.json]: Removed these colors.

File: voter-dapp/README.md (hunk 5 above)
Comment (ptare) [reply voter-dapp/README.md]: Done.

File: package.json (hunk 6 above)
Comment (ptare) [reply package.json]: Yeah `npm start` was complaining about a version mismatch without this update.

**Comment by ptare** [comment]
> I know that this is just the basic react template stuff, but I think this is a good time to cut down or mess with the stuff we rarely touch because there will be a higher activation energy to do so later :).

PTAL.

File: voter-dapp/public/manifest.json (hunk 3 above)
Comment (mrice32) [reply voter-dapp/public/manifest.json]: Sweet. Just wanted to double check.

**Pull request merged**